pine64/rock64/default.nix

```diff
@@ -0,0 +1,26 @@
+{
+  boot = {
+    initrd.kernelModules = [
+      "pcie_rockchip_host"
+      "phy_rockchip_pcie"
+      "sdhci_pci"
+    ];
+    kernelModules = [
```

Comment:
Why is any of this needed? I don't remember needing to specify this.

Reply:
I remember some of them not being detected automatically, but I can remove that `boot.kernelModules` list entirely if it's out-of-scope for nixos-hardware

Reply:
So it won't show any graphics if you attach a monitor, or what does this fix? Usually the kernel will match pci/usb vendor ids with drivers and load them as needed.

Reply:
I am fine with adding modules if they are actually not recognize by the hardware.

Reply:
makes sense, then I'll boot once without this list, and I'll compare which modules are loaded
might take a bit, I won't have access to the SBC for a few days